schemas/taip-0/DepositRequest.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://tap.rspv/schemas/DepositRequest.json",
+  "title": "DepositRequest",
+  "description": "A deposit request",
+  "type": "object",
+  "required": [
+    "transactionId",
+    "amount",
+    "asset",
+    "originators",
+    "beneficiaries",
+    "agents",
+    "settlementInfo"
+  ],
+  "properties": {
+    "transactionId": {
```

Comment:
Unsure if perhaps this should also be a nested JSON-LD `{ "@id": "" }` structure.